pkg/util/cluster.go

```diff
@@ -33,6 +33,7 @@ type ClusterInfo interface {
 	IsControlPlaneHighlyAvailable() bool
 	IsInfrastructureHighlyAvailable() bool
 	IsConsolePluginImageProvided() bool
+	IsWaspImageProvided() bool
```

Comment:
This should not be part of this interface, IMHO (`IsConsolePluginImageProvided` should not be here as well). The interface is about the cluster HCO is running in.

The question if an image is provided or not, does not belong here.

Reply:
@nunnatsa Following our conversation, we decided to defer it to follow up PR